lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -117,4 +120,95 @@ export default class GutenbergEditorHeaderComponent extends AsyncBaseContainer {
 		const element = await this.driver.findElement( titleSelector );
 		return await element.getAttribute( 'value' );
 	}
+
+	async sendFile( name, path ) {
+		const fileNameInputSelector = webdriver.By.css(
+			'.components-form-file-upload input[type="file"]'
+		);
+		const driver = this.driver;
+
+		await driverHelper.waitTillPresentAndDisplayed(
+			driver,
+			By.css( '.components-form-file-upload ' )
+		);
+		let filePathInput = await driver.findElement( fileNameInputSelector );
+		await filePathInput.sendKeys( path );
+
+		await this.openSidebar();
+
+		await driverHelper.clickWhenClickable(
+			driver,
+			By.css( '.components-textarea-control__input' )
+		);
+		let altTextInput = await driver.findElement( By.css( '.components-textarea-control__input' ) );
+		await altTextInput.click();
+		// There's a little time here where the image block seems to refresh, if the alt text is entered before the refresh it will be cleared before saving
+		await this.driver.sleep( 2000 );
+		await altTextInput.sendKeys( name );
+		return await this.closeSidebar();
+	}
+
+	async enterImage( fileDetails ) {
+		const newImageName = fileDetails.imageName;
+		const newFile = fileDetails.file;
+
+		await this.sendFile( newImageName, newFile );
+	}
+
+	async toggleSidebar( open = true ) {
+		const sidebarSelector = '.edit-post-sidebar-header';
+		const sidebarOpen = await driverHelper.isEventuallyPresentAndDisplayed(
```

Comment:
We should use `const sidebarOpen = await driverHelper.isElementPresent(` instead as this implementation waits 30 seconds to determine if it's open